react-app/src/library/services/vendor-implementations/jabra/jabra.ts

```diff
@@ -370,16 +372,29 @@ export default class JabraService extends VendorImplementation {
     this.changeConnectionStatus({ isConnected: true, isConnecting: false });
   }
 
+  async deviceHasPermissions (deviceLabel: string): Promise<boolean> {
+    const allowedHIDDevices = await (window.navigator as any).hid.getDevices();
+    let deviceFound;
+    allowedHIDDevices.forEach(device => {
+      if (deviceLabel.includes(device?.productName?.toLowerCase())) {
+        deviceFound = true;
+        return;
```

Comment:
this doesn't actually work because you can't break out of a `foreach`. The better approach would be to assume you *didn't* find what you wanted until you do. e.x.:

```
    let deviceFound = false;
    allowedHIDDevices.forEach(device => {
      if (deviceLabel.includes(device?.productName?.toLowerCase())) {
        deviceFound = true;
      }
    });
    return deviceFound;
```